source/includes/mongodb-compatibility-table-c.rst

```diff
@@ -4,14 +4,26 @@
    :class: compatibility-large
 
    * - C Driver Version
+     - MongoDB 4.2
      - MongoDB 4.0
      - MongoDB 3.6
      - MongoDB 3.4
      - MongoDB 3.2
      - MongoDB 3.0
      - MongoDB 2.6
 
+   * - Version 1.15
```

Comment:
Was skipping 1.12 through 1.14 (inclusive) intentionally?

In the past, Version 1.8 and Version 1.7 were both included even though they support the same server versions. Would it make more sense to denote a range of versions in the first column (e.g. "Version 1.11 - 1.14") and consolidate rows with identical checkmarks?

Reply:
@jmikola There's no good reason, it just looks like that info was missing. I've updated to include missing versions.

I'm not specifically against doing a version range but it won't be consistent across the rest.